src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
+import ReactDOM from 'react-dom';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
+  document.getElementById('root'),
```

Comment:
- [ ] **[OPTIONAL]:** Please consider removing this line since it's not doing anything here, the render method only receives one argument! 🤓 🔧

Reply:
thanks for your time time